shellbase-slack/src/main/scala/com/sumologic/shellbase/slack/PostCommandWithSlackThread.scala

```diff
@@ -0,0 +1,98 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.sumologic.shellbase.slack
+
+trait PostCommandWithSlackThread extends PostCommandToSlack {
+
+  // add users who have different slack name than local user name: Map(local, slack)
+  protected val diffLocalToSlackUserNameMap: Map[String, String] = Map.empty
+
+  // command execution time threshold that we want to be notified by Slack about command execution result
+  // 5 minutes as default
+  protected val commandNotifyUserTimeThreshold: Long = 5
```

Comment:
Most things that are a long are in milliseconds (at least internally at Sumo.)  We should use `Int` and the variable name should clarify this is `minutes`